packages/terra-form/src/Field.jsx

```diff
@@ -40,24 +52,56 @@ const propTypes = {
    * Determines whether the field is required.
    */
   required: PropTypes.bool,
+  /**
+   * Determines whether or not to append the 'optional' label to a non-required field label.
+   */
+  showOptional: PropTypes.bool,
 };
 
 const defaultProps = {
   children: null,
   error: null,
+  errorIcon: <IconError />,
   help: null,
+  hideRequired: false,
   htmlFor: undefined,
+  isInvalid: false,
   isInline: false,
   label: null,
   labelAttrs: {},
   required: false,
+  showOptional: false,
+};
+
+const contextTypes = {
+  /* eslint-disable consistent-return */
+  intl: (context) => {
+    if (context.intl === undefined) {
+      return new Error('Please add locale prop to Base component to load translations');
```

Comment:
It feels like the error her is actually not wrapping the field inside terra-i18n or terra-base.

Reply:
What do you mean? I'm not sure I understand.

Reply:
Sorry for vaguness.

I think this error comes not from providing a locale to the base component, but by not wrapping this field inside a terra-i18n or react-intl object. For example, if I call on FormattedMessage from react-intl but don't wrap the component inside the IntlProvider, I get this error message

"[React Intl] Could not find required `intl` object. <IntlProvider> needs to exist in the component ancestry."

I feel that the error message should direct users to ensuring they have wrapped this component in terra-base or the terra-i18n-provider. (IMO, I would direct them to ensuring this is wrapped in terra-base, and not even mention the terra-i18n-provider that we have)

Reply:
That's probably something that should be logged as a separate issue seeing as this string has been copy and pasted in all of our components that have translations.